src/exceptions.rs

```diff
@@ -165,7 +199,22 @@ macro_rules! create_exception {
 
         $crate::impl_exception_boilerplate!($name);
 
-        $crate::create_exception_type_object!($module, $name, $base);
+        $crate::create_exception_type_object!($module, $name, $base, ::std::option::Option::None);
```

Comment:
We could also have a default docstring. I chose not to because I'm not sure whether we should write things that may be visible to end users.

Reply:
Yeah I agree that we should not generate a default docstring.